Packs/Whois/Integrations/Whois/Whois.py

```diff
@@ -9086,7 +9086,7 @@ def whois_and_domain_command(command: str, reliability: str) -> list[CommandResu
                         headers=hr_headers,
                         removeNull=True,
                     ),
-                    raw_response=str(domain_data),
+                    raw_response=dict(domain_data),
```

Comment:
Let's fix for all raw responses

Reply:
The other `raw_response` are used in the legacy flow. This is the only place that the new flow returns a raw response. I thought we agreed to make this change only in the new flow?